source/icu.net/icu.net.csproj

```diff
@@ -16,7 +16,7 @@ NOTE: this package contains the managed wrapper part of icu.net. You'll also hav
     <PackageReference Include="GitVersion.MsBuild" Version="5.8.1" PrivateAssets="all" />
     <PackageReference Include="JetBrains.Annotations" Version="2021.3.0" PrivateAssets="all" />
     <PackageReference Include="SIL.ReleaseTasks" Version="2.5.0" PrivateAssets="all" />
-    <PackageReference Include="System.ValueTuple" Version="4.5.0" />
+    <PackageReference Include="System.ValueTuple" Version="4.4.0" />
```

Comment:
Can you please add a comment in code why this should stick with 4.4? Otherwise there's the danger that in a future PR it gets updated again...

Reply:
> Can you please add a line to `CHANGELOG.md`, and `+semver:major` or `+semver:minor` to the commit message (I'm still debating whether it's a major (i.e. breaking) or minor change - any thoughts welcome).
> 
> > As opposed to 4.5, which causes a versioning conflict in repos like FLExBridge. The projects there that target .NET Standard 2.0 also reference v4.4, resulting in a small but ugly warning.
> 
> What speaks against updating System.ValueTuple to 4.5 in FLExBridge et al.? I'm a bit hesitant to downgrade since that could cause problems in other projects.

.NET Standard references v4.4, so no way to get around that. We don't have to upgrade, as it's just a warning, but I was seeing it when consuming from Chorus. There's an ongoing Slack thread with Jason I tagged you in, I'll post the resolution back here